examples/kms/vault/azure-credentials.yaml

```diff
@@ -0,0 +1,10 @@
+---
+# This is an example Kubernetes Secret that can be created in the Kubernetes
+# Namespace where Ceph-CSI is deployed. The contents of this Secret will be
+# used to connect to the Azure Key Vault.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-csi-azure-credentials
+stringData:
+  AZURE_CLIENT_CERTIFICATE: ""
```

Comment:
only the Client certificate is present here and `AZURE_CLIENT_ID` will remain in configmap, which means we will use a single CLIENT_ID which works with multiple certificates.

Reply:
yes, single `CLIENT_ID` can have multiple certificates.